[2023-09-18 22:13:43,712] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,716] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,720] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,723] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,723] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,728] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,748] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,752] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,756] WARNING del_cliente() L128  Erro ao deletar cliente #'undefined', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 22:13:43,762] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "[33mDELETE /deletar_cliente?nome=undefined HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,763] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,768] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,769] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,777] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,777] WARNING del_cliente() L128  Erro ao deletar cliente #'undefined', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 22:13:43,780] WARNING del_cliente() L128  Erro ao deletar cliente #'undefined', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 22:13:43,783] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,784] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,788] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,789] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "[33mDELETE /deletar_cliente?nome=undefined HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,789] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,792] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "[33mDELETE /deletar_cliente?nome=undefined HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,801] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,805] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,808] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "OPTIONS /deletar_cliente?nome=undefined HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,813] WARNING del_cliente() L128  Erro ao deletar cliente #'undefined', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 22:13:43,817] WARNING del_cliente() L128  Erro ao deletar cliente #'undefined', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 22:13:43,818] WARNING del_cliente() L128  Erro ao deletar cliente #'undefined', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 22:13:43,822] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "[33mDELETE /deletar_cliente?nome=undefined HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,824] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "[33mDELETE /deletar_cliente?nome=undefined HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,826] WARNING del_cliente() L128  Erro ao deletar cliente #'undefined', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 22:13:43,827] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "[33mDELETE /deletar_cliente?nome=undefined HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,828] WARNING del_cliente() L128  Erro ao deletar cliente #'undefined', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 22:13:43,829] WARNING del_cliente() L128  Erro ao deletar cliente #'undefined', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 22:13:43,829] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "[33mDELETE /deletar_cliente?nome=undefined HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,836] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "[33mDELETE /deletar_cliente?nome=undefined HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,839] WARNING del_cliente() L128  Erro ao deletar cliente #'undefined', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 22:13:43,842] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "[33mDELETE /deletar_cliente?nome=undefined HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,844] WARNING del_cliente() L128  Erro ao deletar cliente #'undefined', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 22:13:43,844] WARNING del_cliente() L128  Erro ao deletar cliente #'undefined', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 22:13:43,844] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "[33mDELETE /deletar_cliente?nome=undefined HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 22:13:43,851] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 22:13:43] "[33mDELETE /deletar_cliente?nome=undefined HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
